### PR TITLE
Added PackageDeclaration style rule

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

**Comment by schalkms** [comment]
@vanniktech Yes. We have a separate rule for that.

**Comment by vanniktech** [comment]
Which one?

**Comment by schalkms** [comment]
`ConsecutiveBlankLines.kt`

**Comment by vanniktech** [comment]
Interesting `ConsecutiveBlankLines` does not work for me. The following is not getting flagged when running `detektCheck`:

```kotlin
package com.foo.bar


import android.app.Application
```

**Comment by schalkms** [comment]
Well, I see. The threshold for this rule is set to **three**.

**Comment by vanniktech** [comment]
That's what I thought so too but even 4 new lines does not trigger it:

```kotlin
package com.foo.bar




import android.app.Application
```

**Comment by schalkms** [comment]
Interesting. The rule is activated for me.

**Comment by vanniktech** [comment]
Ah might be since its in `formatting` and I'm not using it.

**Comment by schalkms** [comment]
I think I'll check also for two consecutive blank lines in this rule and leave the consecutive format rule as it is.

File: detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/PackageDeclarationStyle.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		if (importList.imports.isNotEmpty()) {
+			checkPackageDeclaration(importList)
+			checkKtElementsDeclaration(importList)
+		}
+	}
+
+	private fun checkPackageDeclaration(importList: KtImportList) {
+		val prevSibling = importList.prevSibling
+		if (prevSibling is PsiWhiteSpace
+				|| prevSibling is KtPackageDirective && prevSibling.text.isNotEmpty()) {
```
Comment (arturbosch) [inline detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/PackageDeclarationStyle.kt]: Can you extract the conditions into functions, it is hard to follow what is checked.
